multicluster/charts/linkerd-multicluster-link/templates/service-mirror.yaml

```diff
@@ -14,6 +14,11 @@ rules:
 - apiGroups: [""]
   resources: ["namespaces"]
   verbs: ["list", "get", "watch"]
+{{- if .Values.enableNamespaceCreation }}
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["create"]
+{{- end}}
```

Comment:
nit: expand the verbs list in the previous rule, and have this conditional only add the item `create`?

Reply:
I considered this, but it felt cleaner to conditionally add a rule rather than conditionally add verbs to a rule.  But I don't feel strongly either way.